### PR TITLE
 best_response qos change

### PR DESCRIPTION
**Pull request by Renbago**
service and qos settings are added.

### CODE REVIEW

**Comment by Renbago** [comment]
@Zarqu0n review

**Comment by Renbago** [comment]
this [branch](https://github.com/Renbago/slam_toolbox/tree/humble-localizationHealth-withservice) include also service @Zarqu0n

**Comment by oguzhankose** [comment]
this seems okey but we need more explanation. 
P.S. Also please avoid unnecessary diffs.

**Comment by Zarqu0n** [comment]
@Renbago can you explain this service.

File: lib/karto_sdk/include/karto_sdk/Mapper.h
Comment (oguzhankose) [inline lib/karto_sdk/include/karto_sdk/Mapper.h:1]: please remove this

**Comment by oguzhankose** [comment]
i will add this to 0.36 version. untill that please re-check it

**Comment by Renbago** [comment]
Sure this pr can wait the new development,

2 Eyl 2024 Pzt 20:49 tarihinde Oguzhan Kose ***@***.***> şunu
yazdı:

**Pull request merged**